collection.sql

```diff
@@ -32,6 +32,32 @@ CREATE TABLE `films` (
   PRIMARY KEY (`id`)
 ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
 
+LOCK TABLES `films` WRITE;
+/*!40000 ALTER TABLE `films` DISABLE KEYS */;
+
+INSERT INTO `films` (`id`, `title`, `release_year`, `my_review`, `bechdel_status`)
+VALUES
+	(1,'Mad Max: Fury Road','2015',5,'PASS'),
+	(2,'The Matrix','1999',5,'PASS'),
+	(3,'The Departed','2006',4,'FAIL'),
+	(4,'Inception','2010',3,'FAIL'),
+	(5,'The Lion King','1994',5,'FAIL');
+
+/*!40000 ALTER TABLE `films` ENABLE KEYS */;
+UNLOCK TABLES;
+
+
+# Dump of table House_Plants
```

Comment:
You probably don't want the house plants table in here

Reply:
Sorted and exported updated db